osv2/bulk.go

```diff
@@ -26,6 +26,9 @@ type BulkRequest struct {
 
 	// Index determines the entire index for the request
 	Index string
+
+	// OnlyErrorResponse determines if the bulk response is parsed all the time or only when errors present in the response
+	OnlyErrorResponse bool
```

Comment:
I would call this `IgnoreResponseItems` or maybe `IgnoreResponseDocuments` to more directly indicate what it's doing